src/theme.js

```diff
@@ -79,3 +64,53 @@ export const colors = {
     light: '#8f9297'
   }
 }
+
+export const breakpoints = [
+  32,
+  48,
+  64,
+  80
+]
+
+export const space = [
+  0,
+  4,
+  8,
+  16,
+  32,
+  64,
+  128,
+]
+
+export const fontSizes = [
+  12,
+  14,
+  16,
+  20,
+  24,
+  32,
+  48,
+  64,
+  72,
+  96
+]
+
+export const weights = [
+  400,
+  700
+]
+
+export const radius = 4
+export const font = `-apple-system, BlinkMacSystemFont, sans-serif`
+export const monospace = '"SF Mono", "Roboto Mono", Menlo, monospace'
```

Comment:
font should be `Roboto, Arial, sans-serif` and monospace should be `'Ubuntu Mono Web', 'Courier New', monospace`